cli_tools/gce_vm_image_import/importer/args.go

```diff
@@ -134,6 +150,9 @@ func (args ImportArguments) validate() error {
 		return fmt.Errorf("-%s and -%s can't be both specified",
 			osFlag, customWorkflowFlag)
 	}
+	if !strings.HasSuffix(args.ScratchBucketGcsPath, args.ExecutionID) {
+		panic("Scratch bucket should have been namespaced with execution ID.")
```

Comment:
If no suffix, the following logic won't be broken. Why don't we allow it? The worst case is it behaves as before. It's not worth failing customers for this reason.

Reply:
It's a programming bug if this panic is reached; so, if someone submits a regression, all of our e2e tests will fail with a stack trace pointing here.